asyncapi/github/service_types.bal

```diff
@@ -76,15 +79,16 @@ public type ReleaseService service object {
     remote function onReleased(ReleaseEvent payload) returns error?;
 };
 
-# Actions related to a label. The action can be one of created, edited or deleted.
+# Triggers when a new event related to labels is received.
```

Comment:
Suggestion `Triggers when a new event related to labels event is received`

Reply:
What do you mean by `LabelEvent`?

Reply:
the event related to the labels. Even in the onCreated( ) method event type is `LabelEvent`. WDYT?